main.js

```diff
@@ -0,0 +1,152 @@
+        origin: 'United States',
+        imageUrl: 'https://goo.gl/Cro3tL'
+      },
+      {
+        itemId: 7,
+        name: 'Bodhran',
+        brand: 'Meinl',
+        price: 129.99,
+        description: 'A frame drum for people who like to jig.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Ireland',
+        imageUrl: 'https://goo.gl/YNb74O'
+      },
+      {
+        itemId: 8,
+        name: 'TR-09',
+        brand: 'Roland',
+        price: 399,
+        description: 'A drum machine for people with no desk space.',
+        details: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+        origin: 'Japan',
+        imageUrl: 'https://goo.gl/gJYUxz'
+      }
+    ]
+  },
+  details: {
+    item: null
+  }
+}
+
+function createElement(tagName, attributes, children) {
+  var $HTMLElement = document.createElement(tagName)
+  for (var key in attributes) {
+    $HTMLElement.setAttribute(key, attributes[key])
+  }
+
+  children.forEach(function (child) {
+    if (child instanceof Node) {
+      $HTMLElement.appendChild(child)
+    }
+    else {
+      $HTMLElement.textContent = child
+    }
+  })
+
+  return $HTMLElement
+}
+
+function currencyFormat(price) {
+  var priceStr = price.toString()
+  if (priceStr[priceStr.length - 3] !== '.') {
+    priceStr += '.00'
+  }
+  return '$' + priceStr
+}
+
+function createCard(item) {
+  return createElement('div', { class: 'card', style: 'width: 18rem;' }, [
+    createElement('img', { class: 'card-img-top', src: item.imageUrl, alt: 'Card image cap' }, []),
+    createElement('div', { class: 'card-body' }, [
+      createElement('h5', { class: 'card-title' }, [item.brand]),
+      createElement('p', { class: 'card-text' }, [item.name]),
+      createElement('p', { class: 'card-text' }, [currencyFormat(item.price)]),
+      createElement('a', { href: '#', class: 'btn btn-primary' }, ['Add to Cart'])
```

Comment:
This `a` doesn't do anything, so you can delete it for now. We try to avoid adding UI elements that are non-functional.